injected/integration-test/autofill-password-import.spec.js

```diff
@@ -0,0 +1,116 @@
+import { test, expect } from '@playwright/test'
+import { readFileSync } from 'fs'
+import {
+    mockAndroidMessaging,
+    wrapWebkitScripts
+} from '@duckduckgo/messaging/lib/test-utils.mjs'
+import { perPlatform } from './type-helpers.mjs'
+
+test('Password import feature', async ({ page }, testInfo) => {
+    const passwordImportFeature = AutofillPasswordImportSpec.create(page, testInfo)
+    await passwordImportFeature.enabled()
+    await passwordImportFeature.navigate()
+    await passwordImportFeature.clickOnElement('Home page')
+    await passwordImportFeature.assertElementIsAnimating('a[aria-label="Password options"')
+
+    await passwordImportFeature.clickOnElement('Signin page')
+    await passwordImportFeature.assertElementIsAnimating('a[aria-label="Sign in"')
+
+    await passwordImportFeature.clickOnElement('Export page')
+    await passwordImportFeature.assertElementIsAnimating('button[aria-label="Export"')
```

Comment:
Will this ordering not be flaky?
Would it be more reliable to have a `waitForAnimate` style function, something like:
```JavaScript
const didAnimateExport = passwordImportFeature.waitForAnimation('button[aria-label="Export"', timeout)
await passwordImportFeature.clickOnElement('Export page')
await didAnimateExport
```

Where `waitForAnimation` could wait for an `animationstart` event.

Reply:
Changed to await after click event: https://github.com/duckduckgo/content-scope-scripts/pull/1157/commits/e18ebbc3e7797139e02c8fc07706e986966dc098